chia/full_node/full_node.py

```diff
@@ -1048,17 +1048,17 @@ async def update_wallets(
         lookup_coin_ids: List[bytes32],
     ) -> None:
         # Looks up coin records in DB for the coins that wallets are interested in
-        new_states: List[Optional[CoinRecord]] = [
-            await self.coin_store.get_coin_record(coin_id) for coin_id in list(lookup_coin_ids)
-        ]
+        new_states: List[CoinRecord] = await self.coin_store.get_coin_records(list(lookup_coin_ids))
+
```

Comment:
if `lookup_coin_ids` isn't a list, wouldn't mypy complain at the type hint in the function signature?

Reply:
yes

Reply:
so the `list()` conversion is probably redundant then